### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by bigs**

### CODE REVIEW

**Comment by parkan** [comment]
Needs a bit about ipfs/`-s` probably, can mostly just link to https://github.com/ipfs/go-ipfs#build-from-source

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# perceptive-client
+
+Lookup image metadata locally using pHash index
+
+## Summary
+
+This is a [mediachain](https://medium.com/mine-labs/mediachain-483f49cbe37a)
+demo that - given an image - will look up its metadata as stored on IPFS. For
```
Comment (parkan) [inline README.md]: I think em dashes are normally `--`

Comment (bigs) [inline README.md]: /me pushes up glasses

in this case the hyphen takes the place of a comma

Comment (parkan) [inline README.md]: "The em dash is perhaps the most versatile punctuation mark. Depending on the context, the em dash can take the place of commas, parentheses, or colons—in each case to slightly different effect."

**Pull request merged**